benchmark/matmul.cpp

```diff
@@ -166,21 +166,13 @@ static void SingleMatmulBase(
         "Shared memory bank conflict not removed.");
   }
 
+  std::vector<c10::IValue> aten_inputs({inputs.first, inputs.second});
+
   // Warm up run
```

Comment:
It looks like we don't have warm up runs and validation of results for other benchmark cases. Do we want to add these to other cases or matmul is the special one?

Reply:
A fusion is compiled and ran once before measurement, so that should be sufficient as a warmup run. 

For validation, that's a good point and I agree that we should have result checking of benchmark outputs. Created an issue here #461